Dockerfile.updater-core

```diff
@@ -88,7 +88,7 @@ WORKDIR $DEPENDABOT_HOME/dependabot-updater
 
 # Install Ruby from official Docker image
 # When bumping Ruby minor, need to also add the previous version to `bundler/helpers/v{1,2}/monkey_patches/definition_ruby_version_patch.rb`
-COPY --from=ruby:3.1.4 --chown=dependabot:dependabot /usr/local /usr/local
+COPY --from=ruby:3.1.4-bullseye --chown=dependabot:dependabot /usr/local /usr/local
```

Comment:
Why is this change needed?

Reply:
I originally created this merge from the point that we took an internal fork.  The latest iteration of my commits doesn't have this change.